cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometry.java

```diff
@@ -0,0 +1,20 @@
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import ucar.ma2.Array;
+
+/**
+ * An interface to interact with
+ * CF 1.8 Compliant Simple Geometries,
+ * and an Enum for the different
+ * geometries
+ *
+ * @author Katie
+ *
+ */
+enum CFGEOMETRY {
+	CFPOINT, CFLINE, CFPOLYGON;
+}
+
+public interface SimpleGeometry {

```

Comment:
It is rare for interfaces to lack methods. Interfaces without methods are sometimes called "marker interfaces". A quick search for this term will reveal that this practice is somewhat controversial. That doesn't mean that we can't use this empty interface, but it is worth evaluating the arguments others have made against marker interfaces. If there are no methods, are we sure we need this interface? Alternatively, Is this interface missing any methods that we should define?

Reply:
added the common methods from the interfaces that extend it